Sources/SmileID/Classes/SmileID.swift

```diff
@@ -57,14 +60,24 @@ public class SmileID {
         return SmileView(initialDestination: destination).environmentObject(navigationState)
     }
 
-    public class func documentVerificationScreen(userId _: String = "user-\(UUID().uuidString)",
-                                                 jobId _: String = "job-\(UUID().uuidString)",
-                                                 showAttribution _: Bool = true,
-                                                 showInstruction _: Bool = true,
+    public class func documentVerificationScreen(userId: String = "user-\(UUID().uuidString)",
+                                                 jobId: String = "job-\(UUID().uuidString)",
+                                                 idType: Document =  Document(countryCode: "",
```

Comment:
We should not default to a `Document` object with an empty countryCode. That is a required value the partner must provide us